src/probnum/problems/_problems.py

```diff
@@ -194,6 +194,26 @@ class LinearSystem:
     # For testing and benchmarking
     solution: Optional[Union[np.ndarray, randvars.RandomVariable]] = None
 
+    @classmethod
+    def from_matrix(
```

Comment:
I feel like this should go into the problem zoo, since there is the specific choice of the normally distributed rhs

Reply:
I hesitated creating a problem zoo method which is really just a convenience constructor for a `LinearSystem`. Now arguably `random_spd_matrix` could then also be called in a constructor for `LinearSystem`. To alleviate the choice of normal distribution I gave `from_matrix` an optional `solution_rv` argument where one can specify a random variable to draw the solution from.

Let me know if you prefer a function under problem zoo which has the equivalent functionality and is called `random_linear_system`, which takes a matrix / callable sampling a system matrix and a randvar / solution to generate the right hand side from.

Reply:
I think I would prefer the `random_linear_system`, but don't feel obliged to change things if this causes inconvenience.